src/stores/authContext.tsx

```diff
@@ -25,6 +31,11 @@ export const AuthContext = createContext<AuthContextType>({
 
 export const AuthProvider = ({ children }: { children: React.ReactNode }) => {
   const [user, setUser] = useState<SessionUser | null>(null)
+  const [portalUser, setPortalUser] = useState<PortalUser | null>(null)
+
+  const setMongoUserInfo = (userData: PortalUser) => {
```

Comment:
I got a bit confused when I saw `setMongoUserInfo` in the other files, mostly because of the naming and it wasn't super clear to me why we needed it.

It makes sense looking at this file, but I think we can simplify it even more and follow the same pattern as `[user, setUser]`

* Change `setMongoUserInfo: (userData: PortalUser) => void` to `setPortalUser: (userData: PortalUser) => void`
* Remove the `setMongoUserInfo` fn entirely
* Add `setPortalUser` to the context const
* In all of the files that import `setMongoUserInfo`, import `setPortalUser` instead, and directly call `setPortalUser`

Is there a drawback to this approach I'm missing? Do we need the extra layer? Lmk what you think!

Reply:
Good catch! Just implemented this. I had initially set it up to use `setPortalUser` directly, but I think I created that function because I was trying to perform another action when we set the user. Just forgot to update it!